### PR TITLE
multi: implement getcfilters rpc.

### PR DESCRIPTION
**Pull request by dnldd**

### CODE REVIEW

File: wire/msggetcfilters.go [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+import (
+	"fmt"
+	"io"
+
+	"github.com/decred/dcrd/chaincfg/chainhash"
+)
+
+// MaxGCFiltersPerMsg is the maximum number of committed filter requests
+// allowed per message.
+const MaxGCFiltersPerMsg = 500
```
Comment (jozn) [inline wire/msggetcfilters.go]: This probably should be lower than 128, every `MsgCFilter` has a max size of 256KB (`MaxCFilterDataSize`), so you don't want to build messages that have more than messages maximum payload size (`MaxMessagePayload` 32MB).

Comment (dnldd) [reply wire/msggetcfilters.go]: good catch, resolving.

File: server.go [hunk 2]
```diff
@@ -931,6 +931,97 @@ func (sp *serverPeer) OnGetCFilter(p *peer.Peer, msg *wire.MsgGetCFilter) {
 	sp.QueueMessage(filterMsg, nil)
```
Comment (jozn) [inline server.go]: I can't comment at the exact line so I leave it in here:

At line 64 `maxProtocolVersion` needs to be changed to the latest wire protocol version.

**Comment by davecgh** [comment]
Now that 1.5.0 is released and version 1 filters are deprecated, I don't think this makes sense.  Closing.

**Pull request closed without merging**